src/xwayland.c

```diff
@@ -279,8 +279,10 @@ handle_request_configure(struct wl_listener *listener, void *data)
 static void
 handle_request_activate(struct wl_listener *listener, void *data)
 {
-	struct view *view = wl_container_of(listener, view, request_activate);
-	assert(view);
+	struct xwayland_view *xwayland_view =
+		wl_container_of(listener, xwayland_view, request_activate);
+	assert(xwayland_view);
```

Comment:
Kind of unrelated to this specific PR but all these asserts after `wl_container_of()` seem superfluous. I actually might have been the one introducing them a while ago.

Reply:
:smile: Removed.